lib/pymedphys/_wlutz/reporting.py

```diff
@@ -20,17 +21,52 @@
 
 
 def image_analysis_figure(
-    x,
-    y,
-    img,
-    bb_centre,
-    field_centre,
-    field_rotation,
-    bb_diameter,
-    edge_lengths,
-    penumbra,
+    x: "np.array",
+    y: "np.array",
+    img: "np.array",
+    bb_centre: Tuple[float, float],
+    field_centre: Tuple[float, float],
+    field_rotation: float,
+    bb_diameter: float,
+    edge_lengths: Tuple[float, float],
+    penumbra: float,
     units="(mm)",
 ):
```

Comment:
go the whole way!  Typing for the output (figure and axes...)

Reply:
Good call. I'll leave this for this one, but will make a habit of it in the future :).